### ISSUE TITLE
Cannot compute intersection between Plane and Line3D

### ISSUE THREAD

**Issue by MarcoMagl** [sympy/sympy#14067]
In the debugging session below, I can compute the intersection between P2 and L_int, but the computation of the intersection between P1 and _Lint raises a ValueError.

ipdb> L_int
Line3D(Point3D(0, 0, 11/20), Point3D(0, 1/20, 11/10))
ipdb> P1
Plane(Point3D(0, 0, 0), (1, 0, 0))
ipdb> P2
Plane(Point3D(0, 0, 11/10), (0, 1, 0))
ipdb> P2.intersection(L_int)
[Point3D(0, 0, 11/20)]
ipdb> P1.intersection(L_int)
*** ValueError: unhandled linear entity: <class 'sympy.geometry.line3d.Line3D'>

**Comment by smichr** [comment]
Check that you are using the most up-to-date SymPy:

```python
>>> import sympy
>>> sympy.__version__
'1.1.2.dev'
>>> p1=Plane(Point3D(0, 0, 0), (1, 0, 0))
>>> p2=Plane(Point3D(0, 0, 11/10), (0, 1, 0))
>>> L=Line3D(Point3D(0, 0, 11/20), Point3D(0, 1/20, 11/10))
>>> [i.intersection(L) for i in (p1,p2)]
[[Line3D(Point3D(0, 0, 11/20), Point3D(0, 1/20, 11/10))], [Point3D(0, 0, 11/20)]]
```

**Comment by gxyd** [comment]
This is even fixed in sympy version 1.0. Probably fixed in PR #11241.
